.github/CONTRIBUTING.md

```diff
@@ -8,8 +8,10 @@ Use `pre-commit install` to install the pre-commit hook for the repository.
 
 - Implement a Formatter by inheriting from `pydocstringformatter.formatting.Formatter`
 - Add your new formatter to `pydocstringformatter.formatting.FORMATTERS`
-- Choose a proper name because this will be user-facing: the name will be used for
-  options of the CLI.
+- Write a clear docstring because this will be user-facing: it's what will be seen in
```

Comment:
I just thought about this. What do we do with the help message if the docstring contains both a summary and body?
Not something to be fixed in this PR perhaps, but we might want to add a `TODO` and open an issue to track.

Reply:
See #28